orchard-provider-aws/src/main/scala/com/salesforce/mce/orchard/io/aws/ProviderSettings.scala

```diff
@@ -7,22 +7,24 @@
 
 package com.salesforce.mce.orchard.io.aws
 
-import com.typesafe.config.Config
-import com.typesafe.config.ConfigFactory
+import scala.util.Try
+import com.typesafe.config.{Config, ConfigFactory}
 
 case class ProviderSettings(config: Config) {
 
-  def loggingUri: Option[String] = {
-    val path = "logging.uri"
-    if (config.hasPath(path)) Option(config.getString(path))
-    else None
-  }
+  def getConfig[T](path: String): Option[T] = if (config.hasPath(path)) {
+    Try(config.getValue(path).unwrapped().asInstanceOf[T]).toOption
```

Comment:
I'm not sure if the type cast would work well with other types like `Int`/`Double`/`Long`

Reply:
@realstraw let me simplify this for now since removal of `enabled` left with all config values of `String` type for now